calicoctl/commands/node/run.go

```diff
@@ -102,6 +103,9 @@ Options:
                            and there are no pre-existing Calico IP pools.
      --disable-docker-networking
                            Disable Docker networking.
+     --docker-networking-ifprefix=<IFPREFIX>
+                           Interface prefix to use for the Docker containers.
```

Comment:
I think this can be beefed up a bit to say docker containers that have been networked using the calico driver.  Also that this is the interface prefix of the interface within the container.